### PR TITLE
Add 8.0.26, 8.1.13 & 8.2.0

### PR DESCRIPTION
**Pull request by mzch**
Add `8.0.26`, `8.1.13` & `8.2.0`
Update xdebug.

### CODE REVIEW

File: share/php-build/definitions/8.2.0 [hunk 1]
```diff
@@ -0,0 +1,10 @@
+configure_option "--enable-gd"
+configure_option "--with-jpeg"
+configure_option "--with-zip"
+configure_option "--with-mhash"
+
+configure_option -D "--with-xmlrpc"
+
+install_package "https://www.php.net/distributions/php-8.2.0.tar.bz2"
+install_xdebug "3.1.6"
```
Comment (GrahamCampbell) [inline share/php-build/definitions/8.2.0:9]: xdebug 3.1.x is not compatible with php 8.2. we need 3.2.0RC2.

**Review by GrahamCampbell — CHANGES_REQUESTED** [review]

File: share/php-build/definitions/8.2.0 (hunk 1 above)
Comment (mzch) [reply share/php-build/definitions/8.2.0]: Thanks @GrahamCampbell  ! fixed.

**Comment by GrahamCampbell** [comment]
Are you able to resolve the merge conflicts, please?

**Comment by GrahamCampbell** [comment]
Can you also move php 8.2.0 to its own PR, unless you are willing to make it work today, but it will require more changes.

**Comment by mzch** [comment]
> Can you also move php 8.2.0 to its own PR, unless you are willing to make it work today, but it will require more changes.

I see. Once I will close this PR.

**Pull request closed without merging**